lib/DxilValidation/DxilValidation.cpp

```diff
@@ -1942,6 +2105,16 @@ static void ValidateDxilOperationCallInProfile(CallInst *CI,
                                  GetLaunchTypeStr(nodeLaunchType)});
 
     break;
+  case DXIL::OpCode::MatVecMul:
+  case DXIL::OpCode::MatVecMulAdd:
+    ValidateImmOperandsForMatVecOps(CI, opcode, ValCtx);
+    break;
+  case DXIL::OpCode::OuterProductAccumulate:
+    ValidateImmOperandsForOuterProdAcc(CI, opcode, ValCtx);
```

Comment:
```suggestion
    ValidateImmOperandsForOuterProdAcc(CI, ValCtx);
```

Reply:
Wasn't able to commit this suggestions but incorporated the change manually. Resolved.